Grand Circus - OOP/GrandCircus/CircusModel/AnimalBase.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace iQuest.GrandCircus.CircusModel
+{
+    internal abstract class AnimalBase : IAnimal
+    {
+        private readonly string name;
+        private readonly string speciesName;
+
+        public AnimalBase(string name,string speciesName)
+        {
+            this.name = name;
```

Comment:
Once we have the get properties, we can directly set them here: Name = name; 
Then, the private fields can be removed.